content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -187,7 +188,18 @@ replace it with `member4=http://10.0.0.4`.
    fd422379fda50e48, started, member3, http://10.0.0.3:2380, http://10.0.0.3:2379
    ```
 
-2. Remove the failed member:
+2. Do either of the following:
+
+   1. If each Kubernetes API server is configured to communicate with all etcd
+      members, remove the failed member from the `--etcd-servers` flag, then
+      restart each Kubernetes API server.
+   2. If each Kubernetes API server communicates with a single etcd member,
+      configure Kubernetes API server clients to not route requests to the
+      Kubernetes API server that communicates with the failed etcd member.
```

Comment:
How could an API server client know which API server instance communicates with which etcd member?

Reply:
You configure it to talk to it locally...

Reply:
Then ... what do you mean by "Kubernetes API server clients"?

Reply:
That's why I suggested this change: https://github.com/kubernetes/website/pull/35882#discussion_r943979925